shipit.yml

```diff
@@ -1,4 +0,0 @@
-ci:
-  hide:
-    -  buildkite/kubernetes-deploy-gem-nightly
```

Comment:
🔥  `.buildkite/pipeline.nightly.yml `  , you'll need to delete the pipeline and probably kill the webhook that triggers it